src/ingameop.cpp

```diff
@@ -183,7 +183,7 @@ static bool addAudioOptions()
 	row++;
 #endif
 
-	addIGTextButton(INTINGAMEOP_GO_BACK, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_SW_W, _("Go Back"), OPALIGN);
+	addIGTextButton(INTINGAMEOP_AUDIO_BACK, INTINGAMEOP_1_X, INTINGAMEOPAUTO_Y_LINE(row), INTINGAMEOP_SW_W, _("Go Back"), OPALIGN);
```

Comment:
I'm sure it works fine but this shouldn't be necessary. A "isAudioOptionsUp" flag with a similar "runIGAudioOptionsMenu" function would be more in line with how the rest of the in-game menus operate.

Reply:
I agree, I rushed this fix because I don't have access to my main pc rn so I wanted to push this asap, anyway I'd rather hold this pr until I can rework this code, it works but I'm not rly happy with my solution..